example-2.0/client/angular-file-upload.js

```diff
@@ -1,3 +1,6 @@
+// Copyright Owner 2016. All Rights Reserved.
```

Comment:
Please update package.json to have `"author": "IBM Corp."` in order for the copyright gets generated properly. Thanks.